Engine/envvar.c

```diff
@@ -1030,12 +1030,13 @@ void *csoundFileOpenWithType(CSOUND *csound, void *fd, int type,
 #if defined(WIN32)
       /* To handle Widows errors in file name characters. */
       size_t sz = 2 * MultiByteToWideChar(CP_UTF8, 0, name, -1, NULL, 0);
-      wchar_t *wfname = alloca(sz);
+      // alloca not available on all platforms
```

Comment:
Don't really need this comment in code if we're removing alloca.

Reply:
The comment was mostly just so alloca doesn't sneak its way back in, but I can remove it too if you want

Reply:
Ok, well, removed the comment